src/main/java/seedu/address/logic/parser/Prefix.java

```diff
@@ -2,7 +2,7 @@
 
 /**
  * A prefix that marks the beginning of an argument in an arguments string.
- * E.g. 't/' in 'add James t/ friend'.
+ * E.g. 'n/' in 'add property n/Alice'.
```

Comment:
Very minor nitpick, but `n/` here should be referring to the property name right? Would it be good to use a property name for the example instead?

Reply:
Ohh right good point. Shall change it. Thanks.